### PR TITLE
Update error code

### LINKED ISSUE #3815: Incorrect error code in limits docs
The limits documentation (https://docs.influxdata.com/influxdb/cloud/account-management/limits/) says you get an error 503 if you exceed your cardinality.  That's not correct. You will get an error 429 (too many requests) when you hit your cardinality limit.

### PR DESCRIPTION
**Pull request by pierwill**
Closes #3815.

### CODE REVIEW

**Comment by pierwill** [comment]
Also, should these fancy quotes be replaced with `"`?

File: content/influxdb/cloud/account-management/limits.md [hunk 1]
```diff
@@ -101,4 +101,4 @@ The following API error responses occur when your plan's service quotas are exce
 | :-----------------------------  | :-----------------------------------------  | :----------- |
 | `HTTP 413 “Request Too Large”`  | cannot read data: points in batch is too large | If a **write** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |  
 | `HTTP 429 “Too Many Requests”`  | Retry-After: xxx (seconds to wait before retrying the request) | If a **read** or **write** request exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) or if a **delete** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |
-| `HTTP 503 “Service unavailable“` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
+| `HTTP 429 “Service unavailable“` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
```
Comment (philjb) [inline content/influxdb/cloud/account-management/limits.md:104]: 429 is also `“Too Many Requests”`

**Comment by jstirnaman** [comment]
> Also, should these fancy quotes be replaced with `"`?

Yep. Fancies were likely pasted in.

File: content/influxdb/cloud/account-management/limits.md (hunk 1 above)
Comment (jstirnaman) [reply content/influxdb/cloud/account-management/limits.md]: @pierwill Does `429` really return a "Service unavailable" message in this case? If so, that's a bug as "Too many requests" is the standard HTTP status message for `429`. If not, then we should add the "Series cardinality..." message to the previous `429` row.

File: content/influxdb/cloud/account-management/limits.md [hunk 2]
```diff
@@ -99,6 +99,6 @@ The following API error responses occur when your plan's service quotas are exce
...
-| `HTTP 503 “Service unavailable“` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
+| `HTTP 413 "Request Too Large"`  | cannot read data: points in batch is too large | If a **write** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |  
+| `HTTP 429 "Too Many Requests"`  | Retry-After: xxx (seconds to wait before retrying the request) | If a **read** or **write** request exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) or if a **delete** request exceeds the maximum [global limit](/influxdb/cloud/account-management/limits/#global-limits) |
+| `HTTP 429 "Service unavailable"` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
```
Comment (jstirnaman) [inline content/influxdb/cloud/account-management/limits.md:104]:
```suggestion
| `HTTP 429 "Too Many Requests"` | Series cardinality exceeds your plan's service quota | If **series cardinality** exceeds your plan's [adjustable service quotas](/influxdb/cloud/account-management/limits/#adjustable-service-quotas) |
```

File: content/influxdb/cloud/account-management/limits.md (hunk 1 above)
Comment (jstirnaman) [reply content/influxdb/cloud/account-management/limits.md]: > @pierwill Does `429` really return a "Service unavailable" message in this case? If so, that's a bug as "Too many requests" is the standard HTTP status message for `429`. If not, then we should add the "Series cardinality..." message to the previous `429` row.

Just realized I was saying the same thing as @philjb.

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**